sqlstore/src/main/resources/db/migration/V0028__model_refactor.sql

```diff
@@ -2,3 +2,28 @@ ALTER TABLE SampleAdditionalInfo ADD COLUMN groupId int(10);
 ALTER TABLE SampleAdditionalInfo ADD COLUMN groupDescription varchar(255);
 
 UPDATE SampleAdditionalInfo SET groupId = (SELECT groupId FROM SampleAnalyte WHERE SampleAdditionalInfo.sampleId = SampleAnalyte.sampleId), groupDescription = (SELECT groupDescription FROM SampleAnalyte WHERE SampleAdditionalInfo.sampleId = SampleAnalyte.sampleId);
+
+ALTER TABLE SampleAdditionalInfo DROP FOREIGN KEY `FKeutn2473w3yr16khgalspuviw`; -- createdBy fkey
+ALTER TABLE SampleAdditionalInfo DROP FOREIGN KEY `FKp8bvx3e7jsmnyw51toi7mq7cq`; -- updatedBy fkey
```

Comment:
The inline comments cause the flyway migration to fail for me. I get the error:
`[ERROR] Message    : You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'ALTER TABLE SampleAnalyte DROP FOREIGN KEY `FKprqyhv40bntjrf5l64mjdgl1j`; -- upd' at line 2`

If the comments occur at the beginning of the line they work okay with flyway. (I'm using flyway via maven.) Changing to something like the following would be fine:

```
-- createdBy fkey
ALTER TABLE Identity DROP FOREIGN KEY `FKauqylg2sle5eudy0tqabtlmsb`; 
-- updatedBy fkey
ALTER TABLE Identity DROP FOREIGN KEY `FKa8c6e56hg9iucguhr0dcse62h`; 
```